zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4995,6 +5053,44 @@ components:
           description: The version of the form.
           type: integer
           format: int64
+    AuthorizationSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/AuthorizationFilterRequest"
+    AuthorizationFilterRequest:
+      type: object
+      properties:
+        ownerType:
+          type: "string"
+    AuthorizationResponse:
+      type: "object"
+      properties:
+        ownerKey:
+          type: "string"
+        ownerType:
+          type: "string"
+        resourceKey:
+          type: "string"
+        resourceType:
+          type: "string"
```

Comment:
can we a shared enum with `AuthorizationPatchRequest.resourceType` https://github.com/camunda/camunda/blob/2c693cd0ecc0a1c984355761f5cacfcda6c2781c/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L3500-L3514